### ISSUE TITLE
Make native install/prefixHeaderFile tasks parallel by default

### ISSUE THREAD

**Issue by ghale** [gradle/gradle-native#67]
Convert the native install and generatePrefixHeaderFile tasks to execute in parallel by default.

**Comment by ghale** [comment]
PR: https://github.com/gradle/gradle/pull/2795

Per our discussion at standup, the prefix header file generation tasks use the worker api to execute in parallel.  It's fairly tricky to construct an integration test that forces another task to run in parallel with these tasks and there is coverage in general to test that worker api tasks execute in parallel, so there is no specific coverage for prefix header file generation tasks (other than the existing tests for these tasks).

**Comment by adammurdoch** [comment]
I wouldn't bother with special coverage for these tasks, they are fast and we can rely on performance tests and the test coverage for the other tasks.

**Comment by ghale** [comment]
This has been merged.
